start.js

```diff
@@ -0,0 +1,5 @@
+const path = require('path');
+const { exec, spawn } = require('child_process');
+
+const result = require('dotenv').config();
+const gatsby = require('gatsby-cli');
```

Comment:
I might be missing something but is this complete?

Reply:
works for me, you may need to create an `.env` file and put one line inside

```
GITHUB_TOKEN=whateverthatis
```

Reply:
According to Gatsby docs, you can just import dotenv in gatsby-config. Anyway we can revisit this in the future if necessary.

Reply:
wow, today i learned